plonk-napi/src/proof.rs

```diff
@@ -1,117 +1,798 @@
-                d
+            #[napi(js_name = [<Wasm $field_name:camel ProofEvaluations>])]
+            #[derive(Clone)]
+            pub struct [<Napi $field_name:camel ProofEvaluations>](
+                ProofEvaluations<PointEvaluations<Vec<$F>>>
+            );
+
+            type NapiProofEvaluations = [<Napi $field_name:camel ProofEvaluations>];
+
+            impl From<NapiProofEvaluations> for ProofEvaluations<PointEvaluations<Vec<$F>>> {
+                fn from(x: NapiProofEvaluations) -> Self {
+                    x.0
+                }
+            }
+
+            impl From<ProofEvaluations<PointEvaluations<Vec<$F>>>> for NapiProofEvaluations {
+                fn from(x: ProofEvaluations<PointEvaluations<Vec<$F>>>) -> Self {
+                    Self(x)
+                }
+            }
+
+            impl FromNapiValue for [<Napi $field_name:camel ProofEvaluations>] {
+                unsafe fn from_napi_value(
+                    env: sys::napi_env,
+                    napi_val: sys::napi_value,
+                ) -> Result<Self> {
+                    let instance = <ClassInstance<[<Napi $field_name:camel ProofEvaluations>]> as FromNapiValue>::from_napi_value(env, napi_val)?;
+                    Ok((*instance).clone())
+                }
+            }
+
+            #[napi(js_name = [<Wasm $field_name:camel LookupCommitments>])]
+            #[derive(Clone)]
+            pub struct [<Napi $field_name:camel LookupCommitments>]
+            {
+                #[napi(skip)]
+                pub sorted: NapiVector<$NapiPolyComm>,
+                #[napi(skip)]
+                pub aggreg: $NapiPolyComm,
+                #[napi(skip)]
+                pub runtime: Option<$NapiPolyComm>,
+            }
+
+            type NapiLookupCommitments = [<Napi $field_name:camel LookupCommitments>];
+
+            #[napi]
+            impl [<Napi $field_name:camel LookupCommitments>] {
+                #[napi(constructor)]
+                pub fn new(
+                    sorted: NapiVector<$NapiPolyComm>,
+                    aggreg: $NapiPolyComm,
+                    runtime: Option<$NapiPolyComm>) -> Self {
+                    NapiLookupCommitments { sorted, aggreg, runtime }
+                }
+
+                #[napi(getter)]
+                pub fn sorted(&self) -> NapiVector<$NapiPolyComm> {
+                    self.sorted.clone()
+                }
+
+                #[napi(getter)]
+                pub fn aggreg(&self) -> $NapiPolyComm {
+                    self.aggreg.clone()
+                }
+
+                #[napi(getter)]
+                pub fn runtime(&self) -> Option<$NapiPolyComm> {
+                    self.runtime.clone()
+                }
+
+                #[napi(setter, js_name="set_sorted")]
+                pub fn set_sorted(&mut self, s: NapiVector<$NapiPolyComm>) {
+                    self.sorted = s
+                }
+
+                #[napi(setter, js_name="set_aggreg")]
```

Comment:
```suggestion
                #[napi(setter, js_name = "set_aggreg")]
```
Just for consistency, sometimes we do leave white spaces like in here and sometimes not